gotham_reqwest/src/main.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) 2020 O.S. Systems Sofware LTDA
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use std::sync::{Arc, Mutex};
+#[macro_use]
+extern crate gotham_derive;
```

Comment:
Is this required? I believe you can use the macro if you import it, as `use gotham::StateData` or similar.